### PR TITLE
release: 9.90-2.29.0

### LINKED ISSUE #6591: Lock wix-style-react
*

### PR DESCRIPTION
**Pull request by jonathanadler**
closes #6591

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6932/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6932

**Comment by mykas** [comment]
Dont forget to add version to wix-style-react changelog too :D

**Comment by jonathanadler** [comment]
> Dont forget to add version to wix-style-react changelog too :D

Added, thanks!

File: packages/wix-ui-tpa/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+* **Dropdown:** Fix default and hover border color  ([1d22f69](https://github.com/wix/wix-style-react/commit/1d22f69602b662b756c5b21ce16f8fe8ffa7716d))
+* **SectionNotification:** Added missing margin in RTL ([8973026](https://github.com/wix/wix-style-react/commit/89730265f3aad7d835aa3449bbcd802e89b2bc61))
+* **TextButton:** Connect the suffix and prefix icons to color and font size ([61876e8](https://github.com/wix/wix-style-react/commit/61876e807c7da69333214cd1dad6f706eae05667))
+* **TextField:** remove Safari extra margin ([#6924](https://github.com/wix/wix-style-react/issues/6924)) ([61b76e8](https://github.com/wix/wix-style-react/commit/61b76e8f91389ca4118cdd195ed2496601fd9ec3))
+
+
+### Features
+
+* **Dropdown:** Add errorTooltipPlacement prop ([443684e](https://github.com/wix/wix-style-react/commit/443684e8f7d819396b80e6f318647a55b7abf3f5))
+
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/CHANGELOG.md:16]: It seems like the following is missing:
```suggestion
**DatePicker:** add `today` prop ([#6911](https://github.com/wix/wix-style-react/pull/6911)) ([e970604](https://github.com/wix/wix-style-react/commit/e9706049a61c73ccc84a15cc50e40fda39457317))
```

Comment (nitayneeman) [reply packages/wix-ui-tpa/CHANGELOG.md:16]: Nevermind it was already released 🤦🏻‍♂️

**Review by mykas — APPROVED** [review]

**Pull request merged**